Source/Libraries/openXDA.APIAuthentication/APIAuthenticationMiddleware.cs

```diff
@@ -36,7 +38,7 @@ namespace openXDA.APIAuthentication
     /// <summary>
     /// Middleware to authenticate users of the API via <see cref="APIAccessKey"/>.
     /// </summary>
-    public class APIAuthenticationMiddleware : OwinMiddleware
+    public partial class APIAuthenticationMiddleware : OwinMiddleware
```

Comment:
Why was this made into a partial class? I don't see any other instances of partial implementations for it.

Reply:
The `ImpersonationSecurityProvider` is actually `private` and inside this class so `ImpersonationSecurityProvider.cs` contains the other `partial` that was to match the `HostSecurityProvider` but I didn't want to add the more extensive `ImpersonationSecurityProvider` in the same File

Reply:
Oh, I see it now. I must have just missed it. Thanks.